docs/input_format.rst

```diff
@@ -71,6 +75,15 @@ Parameters:
 -  ``children``: position of the mesh files for the children surface in
    the FILE section
 
+``WAVE:`` Only applicable in LSPR problems. It specifies physical parameters of
```

Comment:
I think it shouldn't be indented like the others, it's at the same level than `FIELD`. Check how it look on one of the config files from lspr.